whylog/client/const.py

```diff
@@ -1,2 +1,2 @@
 class BufsizeConsts:
```

Comment:
Here is question for @ppolewicz. If namespaces for consts classes should be written with first capital letter?